jobclient/python/tests/test_job.py

```diff
@@ -157,7 +157,7 @@
     'progress_output_file': 'output.txt',
     'progress_regex_string': 'test',
     'gpus': 1,
-    'ports': 10
+    'ports': 10,
```

Comment:
Why the added `,`?

Reply:
ah I was originally adding disk to this job dict but then I removed it to make a more exhaustive test elsewhere. removed the `,`